app/styles/components/_ucsf-layout.scss

```diff
@@ -0,0 +1,17 @@
+@import '~styles/utils/_helpers.scss';
+
+.ucsfHomePage {
+  margin: 100px auto 0;
+  width: 90%;
+}
+
+.title {
+  @extend %font-size-large;
+  color: $color-title;
+}
+
+.subTitle {
+  @extend %font-size-medium;
+  color: $color-grey6;
+  margin: 12px 0 22px;
+}
```

Comment:
> Thanks again for the work on this! I haven't gotten a chance to look over the `UcsfClientEligibilityPage`, but I hope the comments that I've left so far are helpful. Let me know if you have any questions about them.

Thanks! They are very helpful. Thanks for your thoroughness. I have wondered just how granular one should make React components and I'm beginning to see that they should be pretty granular, which I think is great. How about you hold off on reviewing the `UcsfClientEligibilityPage` so that I can apply some of the lessons learned from these comments to that component before you look at it so as to save you some time. I have not been following the the Figma designs with the greatest fidelity, so I will fix that up for both pages and make other changes.